script.js

```diff
@@ -33,33 +28,21 @@ var generateResult = function (userPlays, computerPlays) {
 
   if (userPlays == computerPlays) {
     result = "It's a tie!";
-  }
-
-  if (userPlays == "scissor" && computerPlays == "paper") {
+  } else if (
+    (userPlays == "scissor" && computerPlays == "paper") ||
+    (userPlays == "paper" && computerPlays == "stone") ||
+    (userPlays == "stone" && computerPlays == "scissors")
+  ) {
     result = "You win!";
-    numberOfWins += 1;
-  }
-
-  if (userPlays == "scissors" && computerPlays == "stone") {
-    result = "You lose!";
-  }
-
-  if (userPlays == "paper" && computerPlays == "stone") {
-    result = "You win!";
-    numberOfWins += 1;
-  }
-
-  if (userPlays == "paper" && computerPlays == "scissors") {
-    result = "You lose!";
-  }
-
-  if (userPlays == "stone" && computerPlays == "scissors") {
-    result = "You win!";
-    numberOfWins += 1;
-  }
-
-  if (userPlays == "stone" && computerPlays == "paper") {
+    numberOfWins++;
+    numberOfGames++;
+  } else if (
+    (userPlays == "scissors" && computerPlays == "stone") ||
+    (userPlays == "paper" && computerPlays == "scissors") ||
+    (userPlays == "stone" && computerPlays == "paper")
+  ) {
     result = "You lose!";
+    numberOfGames++;
   }
```

Comment:
I believe you can use an `else` statement for the last condition, since you've already highlighted the win and draw, what's left would be the losing statements